gammapy/estimators/energydependentmorphology.py

```diff
@@ -299,6 +299,21 @@ def run(self, datasets):
         -------
         results : `dict`
             Dictionary with the various energy-dependence estimation values.
+            There are two top level keys ``energy_dependence`` and ``src_above_bkg``.
+
+            ``energy_dependence`` (dict)
```

Comment:
```suggestion
            ``energy_dependence`` (dict): 
            This details the significance of the energy dependence
```

Reply:
This might go in the the introductory sentence: There are two top level keys ``energy_dependence`` (detailing  the morphology energy dependence and its signifcance) and ``src_above_bkg`` (giving significance of the source per energy bin).